nautobot/docs/user-guide/core-data-model/ipam/vlangroup.md

```diff
@@ -3,3 +3,177 @@
 VLAN groups can be used to organize VLANs within Nautobot. Each group may optionally be assigned to a specific location, but a group cannot belong to multiple locations.
 
 Groups can also be used to enforce uniqueness: Each VLAN within a group must have a unique ID and name. VLANs which are not assigned to a group may have overlapping names and IDs (including VLANs which belong to a common location). For example, you can create two VLANs with ID 123, but they cannot both be assigned to the same group.
+
+## VLAN Group Ranges
+
++/- 2.4.0
+
+VLAN Groups contain a mandatory `range` field with a default value of `1-4094` which equals to all VLANs. This field may be used to constrain member VLANs of the group. Only VLANs with VLAN identifiers ("vids") within a `range` value will be allowed to associated with a `VLANGroup`
+
+Range value accepts commas and dashes, with examples as follows:
+
+* 1
+* 1-5
+* 1-5,10
+* 1-5,10-15
+* 1-5,10-15,16,17,18,19,20
+
+Values between dashes will also be expanded into a list of VLANs.
+
+## Creating new VLANs in a VLANGroup programatically
+
++/- 2.4.0
+
+VLAN Groups offer new API endpoint to list available and create new VLANs programatically. Nautobot accepts API calls to the following endpoint:
+
+`/api/ipam/vlan-groups/<vlan_group_id>/available-vlans/`
+
+For the `GET` request, API will return a list of all available VLAN IDs available in the given `VLANGroup`.
```

Comment:
```suggestion
A `GET` request to this API endpoint will return a list of all available (unassigned) VLAN IDs available in the given `VLANGroup`.
```

Reply:
Should we keep the language consistent throughout this doc? `VLAN IDs` vs. `VIDs`